rpcserver.go

```diff
@@ -809,15 +814,151 @@ func (r *rpcServer) VerifyProof(ctx context.Context,
 	)
 	valid := err == nil
 
-	// TODO(roasbeef): also show additional final resting anchor
-	// information, etc?
+	rpcProof, err := marshalProofFile(proofFile, -1)
+	if err != nil {
+		return nil, fmt.Errorf("unable to marshal proof: %w", err)
+	}
 
-	// TODO(roasbeef): show the final resting place of the asset?
-	return &tarorpc.ProofVerifyResponse{
-		Valid: valid,
+	return &tarorpc.VerifyProofResponse{
+		Valid:        valid,
+		DecodedProof: rpcProof,
 	}, nil
 }
 
+// DecodeProof attempts to decode a given proof file that claims to be anchored
```

Comment:
q: is there any difference between what we call "raw poofs" and "proof files"? are they both supposed to be "raw proof in bytes"?

Reply:
Proof file includes a genesis_point for some reason, so I guess "raw proof" is more correct than "proof file"

Reply:
Hmm nevermind actually.. proof file is used here for consistency with the already existing VerifyProof comment

```
// VerifyProof attempts to verify a given proof file that claims to be anchored
// at the specified genesis point.
```

So I think the only difference is that a proof file also has a string field for genesis_point, but weirdly, I don't notice that string value _ever_ being populated anywhere

I also see a todo in verifier.go:
```
// TODO(roasbeef): pass in the expected genesis point here?
```

Reply:
@Roasbeef thoughts here if genesis_point should be included?